boards/xtensa/intel_s1000_crb/doc/intel_s1000.rst

```diff
@@ -218,10 +218,98 @@ However, we need to short 2 pins on Host Connector J3 via a 3.3k resistor
 (simple shorting without the resistor will also do) for debugging to work.
 Those 2 pins are Pin5 HOST_RST_N_LT_R) and Pin21 (+V_HOST_3P3_1P8).
 
+MCUbootloader Support
+=====================
+
+Intel S1000 on Zephyr supports the open source, cross-RTOS MCUbootloader.
+MCUbootloader in conjuntion with USB DFU (Device Firmware Upgrade)
+over the air can be used for Field upgradation of Intel S1000 Firmware.
+The MCUbootloader source code itself is hosted in the
+`MCUboot GitHub repo`_ page. Check out `MCUboot with Zephyr`_ documentation
+page on the MCUboot website for more on MCUboot.
+
+First, you need to generate the MCUbootloader image for Intel S1000.
+Go to the mcubootloader repository on your local machine and execute the
+below commands.
+
+.. code-block:: console
+
+   cd boot/zephyr
```

Comment:
This block is similar to the app build and should be able to use the `zephyr-app-commands` directive. See around line 136

Reply:
zephyr-app-commands are applicable only to those apps which are within the zephyr tree. mcubootloader is outside of it.

Reply:
see `:app:` directive (vs `:zephyr-app:`) at https://docs.zephyrproject.org/latest/documentation/doc-guidelines.html#zephyr-app-commands-directive

Reply:
modified as suggested.